diff --git a/packages/BUILD.bazel b/packages/BUILD.bazel
index c798707..c42bf1a 100644
--- packages/BUILD.bazel
+++ packages/BUILD.bazel
@@ -2,6 +2,22 @@ load(":packages.bzl", "latex_package")
 
 # Externally usable library targets.
 
+latex_package(
+    name = "cite",
+    srcs = [
+        "@texlive_texmf__texmf-dist__tex__latex__cite",
+    ],
+    tests = [],
+)
+
+latex_package(
+    name = "textcomp",
+    srcs = [
+        "@texlive_texmf__texmf-dist__tex__latex__textcomp",
+    ],
+    tests = [],
+)
+
 latex_package(
     name = "adjustbox",
     srcs = [
diff --git a/texlive_2022_repos.bzl b/texlive_2022_repos.bzl
index 151e001..1e9f7e5 100644
--- texlive_2022_repos.bzl
+++ texlive_2022_repos.bzl
@@ -25,6 +25,7 @@ TEXLIVE_MODULAR_PACKAGES_BIN_2022 = [
 
 TEXLIVE_MODULAR_PACKAGES_OTHER_2022 = [
     ("extra/tlpkg/TeXLive", "1e220578cbe21cca2175e68c84e0eefdf64c2ca71ce157e505b23940a337a789", []),
+    ("texmf/texmf-dist/tex/latex/textcomp", "", []),
     ("texmf/texmf-dist/asymptote", "a9a1819f80f75d2c3dfe34b395215fb95b5bfc224c7ab5794f027b788d835334", []),
     ("texmf/texmf-dist/bibtex/bib/abntex2", "c14b514b9e891c4524a1f35f4b7eb2ef75f45c5ece7f41dfb6fd9bb950e29a97", []),
     ("texmf/texmf-dist/bibtex/bib/abstyles", "c34b0359b0a520717fab39f72d14691f597a76fec3987e166b4adbe3100f88b7", []),
